api/candlepin-api-spec.yaml

```diff
@@ -500,6 +636,39 @@ components:
             principalName:
               type: string
 
+    DistributorVersionDTO:
+      description: A DTO representation of the DistributorVersion
+      allOf:
+        - $ref: '#/components/schemas/TimestampedEntity'
+        - type: object
+          required:
+            - name
+          properties:
+            id:
+              type: string
+            name:
+              type: string
+              minLength: 1
```

Comment:
Same here, only the capability name field is validated for being empty, not the distributor verison name

Reply:
Done.